src/accountsdb/db.zig

```diff
@@ -1072,24 +1148,67 @@ pub const AccountsDB = struct {
                 }
 
                 // flush the slots
+                try unclean_account_files.ensureTotalCapacityPrecise(flush_slots.items.len);
                 for (flush_slots.items) |flush_slot| {
-                    self.flushSlot(flush_slot, &unclean_account_files) catch |err| {
+                    const unclean_file_id = self.flushSlot(flush_slot) catch |err| {
                         // flush fail = loss of account data on slot -- should never happen
                         self.logger.errf("flushing slot {d} error: {s}", .{ flush_slot, @errorName(err) });
+                        continue;
                     };
+                    unclean_account_files.appendAssumeCapacity(unclean_file_id);
                 }
+                self.largest_flushed_slot.store(root_slot, .seq_cst);
```

Comment:
probs safter to store the max slot value in `flush_slots.items`

Reply:
I'm not sure I understand what you mean? Could you elaborate?

Reply:
for (flush_slots) |slot| { max_flushed_slot = max(slot, max_flushed_slot) }
self.largest_flushed_slot = max_flushed_slot

Reply:
dacb0ec60e0b84d06474dc121019f053807858ee